packages/ember-truth-helpers/src/helpers/and.ts

```diff
@@ -9,11 +8,11 @@ export interface AndSignature {
   Return: boolean;
 }
 
-export default helper<AndSignature>((params) => {
+export default function and<T extends MaybeTruth[]>(...params: [...T]) {
   for (let i = 0, len = params.length; i < len; i++) {
     if (truthConvert(params[i]) === false) {
       return params[i] as boolean;
     }
   }
   return params[params.length - 1] as boolean;
```

Comment:
keeping the `as boolean`'s here might not quite meet the type needs of https://github.com/jmurphyau/ember-truth-helpers/pull/185 for the and helper

Reply:
(not that this PR needs to solve that too)

Reply:
yeah, I'd prefer to add tests for the scenarios in #185 -- but in a separate PR -- since this PR is mostly a mechanical change branched off of #185, I'd prefer to iterate, so that PRs can remain reviewable